source/includes/options-mongorestore.yaml

```diff
@@ -239,6 +240,11 @@ description: |
   level of the dump directory for :program:`mongorestore --oplogReplay
   <mongorestore>` to replay.
 
+  :program:`mongorestore --oplogReplay <mongorestore>` must restore the
```

Comment:
I think this is the other way around. ":program:`mongorestore --oplogReplay <mongorestore>` requires a full database dump of a :term:`replica set` member"

In fact, it might specifically require the `--oplog` flag to mongoexport. 

Then you can continue with ":program:`mongorestore --oplogReplay <mongorestore>` can only perform full database restorations. You cannot limit the restoration process using options such as --database or --collection